src/test/java/de/tum/in/www1/artemis/AssessmentComplaintIntegrationTest.java

```diff
@@ -429,6 +440,8 @@ private void checkIfNoSensitiveExerciseDataPresent(Complaint receivedComplaint)
             assertThat(exercise.getStudentParticipations()).as("Exercise only contains title and ID").isNullOrEmpty();
             assertThat(exercise.getTutorParticipations()).as("Exercise only contains title and ID").isNullOrEmpty();
             // TODO check exercise type specific sensitive attributes
+            assertThat(modelingExercise.getSampleSolutionModel()).as("Exercise only contains title and ID").isNull();
```

Comment:
Shouldn't the exercise be taken from the passed in `receivedComplaint` that is the result of a REST call? As far as I can see, `modelingExercise` is simply taken from the database and could thus contain very different information than `exericse` in the lines above. I think something along those lines would make more sense (I didn't test it but you get the idea):

```java
if (exercise instanceof ModelingExercise) {
  ModelingExercise modelingExercise = (ModelingExercise) exercise;
  assertThat(modelingExercise.getSampleSolutionModel()).as("Exercise only contains title and ID").isNull();
}
```